.github/workflows/trunk.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+
+on:
+  push:
```

Comment:
It means there is no build after a merge. is it what you want?

Reply:
good catch. Thank you